src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -538,6 +543,11 @@ public AdminUI getAdmin() {
         "kk+spamtest@kohsuke.org"
     );
 
+    public static final List<String> IP_BLACKLIST = Arrays.asList(
+        "122.177.170.96",
```

Comment:
These are repeat offenders I collected from the emails sent to the admin list. The cookie method has caught them, but they are persistent.